File: challengeDevelopment/jsDevChallenges/bl_randomNumberGame.js
function getRandomNumber(max, min) {
    return Math.floor(Math.random() * (max - min + 1) + min);
}

let inputNumber = 4;
let randomNumber = getRandomNumber(1, 100);
let message = "";
let text = "";
    text += inputNumber > randomNumber ? "mayor" : "";
    text += inputNumber < randomNumber ? "menor" : "";

if (inputNumber === randomNumber) {
    message = "El numero ingresado es igual al numero generado (¡Ganaste!)";
} else {
    message = `El numero ingresado es ${text} al numero generado`;
}

console.log("Numero genarado: " + randomNumber);
console.log(message);

File: challengeDevelopment/pyDevChallenges/bl_randomNumberGame.py
from random import *

while True:
    print("\n// Adivina el número //\n")
    print("¡Adivina el numero del 1 al 100!\n")

    inputNumber = int(input("Ingresa el numero: "))
    randomNumber = randint(1, 100)

    if inputNumber > randomNumber:
        text = "mayor"
    elif inputNumber < randomNumber:
        text = "menor"

    if inputNumber == randomNumber:
        message = "El numero ingresado es igual al numero generado (¡Ganaste!)"
    else:
        message = f"El numero ingresado es {text} al numero generado"

    print("Numero generado: " + str(randomNumber))
    print(message)

    exitGame = input("\nPara finalizar el juego precione 'q' o si no cualquier otra tecla: ")

    if exitGame.lower() == "q":
        print("¡Juego finalizado!")
        break